.gitignore

```
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.env
```

Ett tips bara, det kan vara bra att lägga till .env.local också för lokala .env filer, som skiljer sig från .env som går till produktion / live webb

```suggestion
.env
.env.local
```